concordium-consensus/src/Concordium/Scheduler.hs

```diff
@@ -1204,7 +1207,10 @@ handleContractUpdateV1 originAddr istance checkAndGetSender transferAmount recei
       withToContractAmountV1 sender istance transferAmount $ do
         foreignModel <- getRuntimeReprV1 model
         artifact <- liftLocal $ getModuleArtifact (GSWasm.miModule moduleInterface)
-        go [] =<< runInterpreter (return . WasmV1.applyReceiveFun artifact cm receiveCtx receiveName useFallback parameter transferAmount foreignModel)
+        let maxParameterLen = Wasm.maxParameterLen $ protocolVersion @(MPV m)
+        -- Find setting for whether the number of logs and the size of return value should be limited.
```

Comment:
Extremely nitpicky sorry. But I'm not sure "Find setting .." is the best way to phrase this. 
It's not really a `setting' to me (I would at least read this as something that could be toggled on/off/manipulated. 

I would probably vote for simply having the function called `isLogsAndReturnValuesLimited` and then the function is pretty much self explanatory from the context where you're providing the PV.

Reply:
I changed the comment and called it a day ;)